Merge Conflict/Assets/Scripts/UiManager.cs

```diff
@@ -269,12 +301,43 @@ private void SetPlayFieldSpritesVisible(bool isVisible)
 
     private void PauseGame()
```

Comment:
PauseGame and ContinueGame each just call a single function. As that function call describes very well what the function is doing, i think that an extra method layer is not needed. 
Just call GamePauseManager.Instance.Pause(); instead of PauseGame()